### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by gityihang** [nkicsl/Resfusion#13] [Q&A] [answered]
In section 2.2, you mentioned that the true value x0 is not available. May I ask why it cannot be used in the forward process

**Comment by gityihang** [comment]
I have reviewed your code again and it seems that you only mean that the true ground x0 cannot be used in the reverse process

**Reply by YukinoshitaLove** [reply, marked as answer]
Yes. For the forward process, x_0 and R are available. For the reverse process,  x_0 and R are not available since we could not know the gt. `ฅ^•ﻌ•^ฅ`

**Reply by YukinoshitaLove** [reply]
![image](https://github.com/user-attachments/assets/337a4e21-a846-41ff-a51e-8e19368021e1)
